fermipy/gtanalysis.py

```diff
@@ -4024,54 +4024,62 @@ def get_src_model(self, name, paramsonly=False, reoptimize=False,
             thesrc = self.like[name]
             src_dict['flux'] = self.like.flux(name, self.energies[0],
                                               self.energies[-1])
-            src_dict['flux100'] = self.like.flux(name, 100., 10 ** 5.5)
-            src_dict['flux1000'] = self.like.flux(name, 1000., 10 ** 5.5)
-            src_dict['flux10000'] = self.like.flux(name, 10000., 10 ** 5.5)
             src_dict['eflux'] = self.like.energyFlux(name,
                                                      self.energies[0],
                                                      self.energies[-1])
-            src_dict['eflux100'] = self.like.energyFlux(name, 100.,
-                                                        10 ** 5.5)
-            src_dict['eflux1000'] = self.like.energyFlux(name, 1000.,
-                                                         10 ** 5.5)
-            src_dict['eflux10000'] = self.like.energyFlux(name, 10000.,
-                                                          10 ** 5.5)
-            src_dict['dnde'] = self.like[name].spectrum()(
-                pyLike.dArg(src_dict['pivot_energy']))
-            src_dict['dnde100'] = self.like[name].spectrum()(
-                pyLike.dArg(100.))
-            src_dict['dnde1000'] = self.like[name].spectrum()(
-                pyLike.dArg(1000.))
-            src_dict['dnde10000'] = self.like[name].spectrum()(
-                pyLike.dArg(10000.))
+            emax = np.min([self.energies.max(), 10 ** 5.5])
```

Comment:
Since the two values are in a list, you might considering using `min` here, instead of `np.min`, since the latter needs to convert your list to a `np.ndarray` first and then take the minimum, while the former is a built in and do not need any conversion.